docs/TBG_macros.cfg

```diff
@@ -124,7 +124,8 @@ TBG_stopWindow="--stopWindow 1337"
 #--<species>_radiation.end     Time step to stop calculating the radiation
 #--<species>_radiation.radPerGPU     If flag is set, each GPU stores its own spectra without summing the entire simulation area
 #--<species>_radiation.folderRadPerGPU     Folder where the GPU specific spectras are stored
-#--e_<species>_radiation.compression    If flag is set, the hdf5 output will be compressed.
+#--<species>_radiation.compression    If flag is set, the hdf5 output will be compressed.
+#--<species>_radiation.numTmpResults Number of temporary results per device.
```

Comment:
I don't like `numTmpResults` as this feels very unclear for a user. I suggest using word `granularity`, as this more clearly represents the meaning, and also is used in the same meaning for general parallel processing.

Reply:
I think `granularity` is confusing here. On first reading I would _not_ attribute the CS task granularity to it, when trying to set up a physics simulation.
Maybe something like `numParAccumulations` ?

Reply:
`numParAccumulations` must be `numParallelAccumulations` because `par` is the common used abbreviation  in PIConGPU for `particle`

Reply:
In general the arrays are `intermediate` results. The naming should more what it is instead where or how it is used.

Reply:
I changed it now to `numJobs` what is nearly similar to the option `j` in `make`

Reply:
I like @psychocoderHPC choice - alternatively how about `numAccumulationTasks`?

Reply:
IMO `numAccumulationTasks` is to near to what we are doing. There is no need for the user (in the short name) to know how it is implemented.